front/src/Connexion.ts

```diff
@@ -166,16 +166,15 @@ export class Connexion implements ConnexionInterface{
 
     /**
      *
-     * @param roomId
      * @param x
      * @param y
      * @param direction
      */
-    sharePosition(roomId : string, x : number, y : number, direction : string = "none") : void{
+    sharePosition(x : number, y : number, direction : string = "none") : void{
         if(!this.socket){
             return;
         }
-        let messageUserPosition = new MessageUserPosition(this.userId, roomId, new Point(x, y, direction));
+        let messageUserPosition = new MessageUserPosition(this.userId, ROOM[0], new Point(x, y, direction));
```

Comment:
We should make sure the room is completely removed from the `MessageUserPosition` message.

Instead, when a user joins a room (when he/she enters the GameScene), we should send a `connectToMap` message containing the name of the room we want to join (bonus: the server should reply with the URL of the Tiled map to download).

Reply:
The `connectToMap` message should be sent by the GameManager I guess.

Reply:
I am simply removing the concept of room for now. I will change the scope of gameManager in another PR